accelerators-home/modules/ROOT/pages/simplifying-assets.adoc

```diff
@@ -0,0 +1,66 @@
+= Considerations for Simplifying Accelerator Assets
+
+MuleSoft Accelerators were designed following an API-led approach to connectivity, which promotes a clean separation of application, process, and system integration layers while enabling a high level of reusability - particularly with System APIs. While this powerful approach remains valid today, and is highly recommended for organizations of all sizes to enable future growth while minimizing maintenance costs, there are some considerations that need to be taken into account for customers who want to simplify their integrations and architectural approach. 
```

Comment:
To avoid the passive and extra words, suggest something like:

MuleSoft Accelerators follow an API-led approach to connectivity, which promotes a clean separation of application, process, and system integration layers while enabling a high level of reusability, particularly with System APIs. While this powerful approach remains valid and recommended for organizations of all sizes to enable future growth while minimizing maintenance costs, consider the following guidelines to simplify your integrations and architectural approach.

Reply:
Did you run this through vale? The sentence length and passive voice should be caught by vale, I think.